lib/rake/install.rake

```diff
@@ -53,7 +53,7 @@ namespace :install do
             # cache, prepare one from scratch by creating the database and loading
             # the schema. If we do, make sure it's up to date with the branch
             # being tested. Either way, seeding will be performed in a later step.
-            RakeUtils.rake 'db:setup_or_migrate'
+            RakeUtils.rake_stream_output 'db:setup_or_migrate'
```

Comment:
make it so that we can see when a migration is running in drone. example output: 
```
94 | RAILS_ENV=test RACK_ENV=test bundle exec rake db:setup_or_migrate
...
99 | == 20250624183835 AlterDashboardUtf8mb4: migrating ============================
100 | -- execute("ALTER DATABASE dashboard_test CHARACTER SET = utf8mb4 COLLATE = utf8mb4_unicode_ci;")
101 | -> 0.0923s
102 | == 20250624183835 AlterDashboardUtf8mb4: migrated (0.0924s) ===================
...
105 | Finished db:setup_or_migrate (53 seconds)
```